src/pages/outcomes/cost-and-carbon-savings.jsx

```diff
@@ -0,0 +1,220 @@
+import Hero from 'components/pages/industries/hero';
+import IndustryUseCases from 'components/pages/industries/industry-usecase';
+import Testimonial from 'components/pages/industries/testimonial';
+import UseCaseCard from 'components/pages/use-cases/use-case-card';
+import Community from 'components/shared/community';
+import Heading from 'components/shared/heading';
+import SEO from 'components/shared/seo';
+import EccoLogo from 'icons/ecco.inline.svg';
+import KakaoLogo from 'icons/kakao.inline.svg';
+import MeltwaterLogo from 'icons/logo-meltwater.inline.svg';
+import SeznamLogo from 'icons/logo-seznam.inline.svg';
+import SmartnewsLogo from 'icons/smartnews.inline.svg';
+import SavingsImage1 from 'images/pages/outcomes/cost-and-carbon-savings/cc-image1.png';
+import SavingsImage2 from 'images/pages/outcomes/cost-and-carbon-savings/cc-image2.png';
+import SavingsBee from 'images/pages/outcomes/cost-and-carbon-savings/savings-bee.png';
+import MainLayout from 'layouts/main/main';
+
+const heroContent = {
+  heading: 'Cost and Carbon Savings',
+  tagline: 'Do more with less',
+  texts: [
+    'Manual and legacy network systems often come with hidden overhead like inefficient IP usage, many network redirects, and cloud charges that add up fast. Cilium removes these inefficiencies with a streamlined, eBPF-powered network engine, lowering costs while reducing compute waste and carbon dioxide emissions.',
+  ],
+  imageSrc: SavingsBee,
+  imageAlt: 'savings bee',
+};
+
+const EccoTestimonial = {
+  logo: 'ecco',
+  description:
+    'ECCO’s IT infrastructure, nearly 100% Kubernetes-based, is designed to facilitate machine learning (ML) workflows that enable intelligent decision making and supply chain management.',
+  quotedText: [
+    'Our new storage system and networking setup powered by Cilium provided up to a 50% reduction in storage operational cost, and it’s all thanks to Cilium enabling a more efficient network. Even with IPv6 networks, enabling Cilium was just a flip of a switch. It was surprisingly straightforward.',
+  ],
+  withPerson: true,
+  name: 'George Zubrienko',
+  role: 'Data & AI lead platform engineer, ECCO',
+  url: 'https://www.cncf.io/case-studies/ecco/',
+  CTAtext: 'Read The Case Study',
+};
+
+const KakaoTestimonial = {
+  logo: 'kakao',
+  description:
+    'Kakao Corp is a South Korean company known for its popular messaging application, KakaoTalk, that serves over 50 million users. They not only help people stay connected but also offer additional services such as maps, comics, online shopping, banking, taxis, and more.',
+  quotedText: [
+    'As an engineer, I can say that Cilium has lowered our costs for performance and network. Because we have so many clusters and nodes, we always need more machines for new services to be served. By reducing network costs and CPU consumption with Cilium, we can manage with fewer nodes. Compared to kube-proxy, Calico, or Flannel, Cilium offers more value in that aspect.',
```

Comment:
```suggestion
    'As an engineer, I can say that Cilium lowered our costs for performance and networking. Because we have so many clusters and nodes, we always need more machines for new services to be served. By reducing network costs and CPU consumption with Cilium, we can manage with fewer nodes. Compared to kube-proxy, Calico, or Flannel, Cilium offers more value.',
```